paper.md

```diff
@@ -0,0 +1,190 @@
+---
+title: 'Introduction to deep learning: Carpentries-style hands-on lesson material for introducing researchers to deep learning'
+tags:
+  - Python
+  - deep learning
+  - machine learning
+  - Keras
+  - neural networks
+authors:
+  - name: Sven A. van der Burg
+    orcid: 0000-0003-1250-6968
+    affiliation: 1 # (Multiple affiliations must be quoted, like "1, 2")
+  - name: Anne Fouilloux
+    orcid: 0000-0002-1784-2920
+    affiliation: 2
+  - name: Florian Huber
+    orcid: 0000-0002-3535-9406
+    affiliation: "1, 3"
+  - name: Dafne van Kuppevelt
+    affiliation: 1
+  - name: Peter Steinbach
+    orcid: 0000-0002-4974-230X
+    affiliation: 4
+  - name: Berend Weel
+    orcid: 0000-0002-9693-9332
+    affiliation: 1
+  - name: Colin Sauze
+    orcid: 0000-0001-5368-9217
+    affiliation: 5
+  - name: Samantha Wittke
+    orcid: 0000-0002-9625-7235
+    affiliation: "6,7"
+  - name: Djura Smits
+    orcid: 0000-0003-4096-0260
+    affiliation: 1
+  - name: Cunliang Geng
+    orcid: 0000-0002-1409-8358
+    affiliation: 1
+  - name: Pranav Chandramouli
+    orcid: 0000-0002-7896-2969
+    affiliation: 1
+
+affiliations:
+ - name: Netherlands eScience Center, Amsterdam, The Netherlands
+   index: 1
+ - name: Simula Research Laboratory, Oslo, Norway
+   index: 2
+ - name: Düsseldorf University of Applied Sciences, Düsseldorf, Germany
+   index: 3
+ - name: Helmholtz-Zentrum Dresden-Rossendorf, Dresden, Germany
+   index: 4
+ - name: National Oceanography Centre, Liverpool, Great-Britain
+   index: 5
+ - name: CSC - IT center for Science, Espoo, Finland
+   index: 6
+ - name: Aalto University, Espoo, Finland
+   index: 7
```

Comment:
```suggestion
  - name: Toby Hodges
    orcid: 0000-0003-1766-456X
    affiliation: 8

affiliations:
 - name: Netherlands eScience Center, Amsterdam, The Netherlands
   index: 1
 - name: Simula Research Laboratory, Oslo, Norway
   index: 2
 - name: Düsseldorf University of Applied Sciences, Düsseldorf, Germany
   index: 3
 - name: Helmholtz-Zentrum Dresden-Rossendorf, Dresden, Germany
   index: 4
 - name: National Oceanography Centre, Liverpool, Great-Britain
   index: 5
 - name: CSC - IT center for Science, Espoo, Finland
   index: 6
 - name: Aalto University, Espoo, Finland
   index: 7
 - name: The Carpentries, USA
   index: 8 
```

Reply:
Adding myself so you don't have to dig up my orcid. Not sure if we have a preferred system for the order of the author list but you are welcome to move me elsewhere however you like.

Reply:
Ah, already added you in the meantime, but I had located the carpentries in Heidelberg instead of the USA. Fixed that with 1fe236a17f0277e863f403fac94c195eaf96bba3